commands/root.go

```diff
@@ -38,8 +38,8 @@ func init() {
 	}
 	rootCmd.PersistentFlags().StringVarP(&rootDir, "rootDir", "r", dir, "Root directory where monitors yml files")
 	rootCmd.PersistentFlags().StringVarP(&esURL, "esUrl", "e", "https://localhost:9200/", "URL to connect to Elasticsearch")
-	rootCmd.PersistentFlags().StringVarP(&userName, "username", "u", "admin", "Username for opendistro Elasticsearch")
-	rootCmd.PersistentFlags().StringVarP(&password, "password", "p", "admin", "Password for opendistro Elasticsearch")
+	rootCmd.PersistentFlags().StringVarP(&userName, "username", "u", "", "Username for opendistro Elasticsearch")
```

Comment:
The overall idea ODFE comes default with security plugin, do you have any other use case where it could be blank ?

Reply:
I'm using Amazon ES with source IP address restriction.

Reply:
I would prefer to pass empty or we could create condition for AES?